tsconfig.json

```diff
@@ -7,11 +7,6 @@
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true
   },
-  "include": [
-    "packages/**/*.ts"
-  ],
-  "exclude": [
-    "node_modules/**",
-    "packages/*/node_modules/**"
-  ]
+  "include": ["packages/**/*.ts"],
+  "exclude": ["node_modules/**", "packages/*/node_modules/**", "packages/*/__tests__/**"]
```

Comment:
We don't want to build the files, so for now I added this match to exclude them

Reply:
Okay!